### PR TITLE
Update README with deployment info, etc

### LINKED ISSUE #9: Support some deployment options out of the box
We should support static deployment platforms and show some examples that can immediately be forked / deployed. 

I’ll list a couple but we should probably just ask different people to PR different providers. eG @bmann can do the Heroku version 

We already support netlify, we should definitely consider IPFS-based providers as a deployment target

TODOs:

- Cloudflare Pages
- Vercel

Maybe in the future:

- Fleek
- Render
- Railway

Fission server (for dnslink) combined with Web3storage or Infura might be interesting to look at.

### PR DESCRIPTION
**Pull request by jessmartin**
# Description

Updating the README.

## Link to issue

- [x] Add Deployment instructions for Vercel and Cloudflare Pages
- [x] Add "experimental software warning"
- [x] Add supported node versions.

Closes #9.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+[![Built by FISSION](https://img.shields.io/badge/⌘-Built_by_FISSION-purple.svg)](https://fission.codes) [![Built by FISSION](https://img.shields.io/badge/webnative-v0.34.1-purple.svg)](https://github.com/fission-suite/webnative) [![Discord](https://img.shields.io/discord/478735028319158273.svg)](https://discord.gg/zAQBDEq) [![Discourse](https://img.shields.io/discourse/https/talk.fission.codes/topics)](https://talk.fission.codes)
 
 ![Webnative UI Screenshot](static/webnative-ui.png)
 
 The Webnative App Template is a clone-and-go template for building a web application using Webnative, fast. Clone, customize, and deploy to have a running distributed app in mere minutes.
 
+<div style="background-color: #FEEB80; padding: 1px 20px; border-radius: 10px;">
```
Comment (jessmartin) [inline README.md:9]: This is not displaying in the diff properly. It actually looks something like this:
image

**Comment by jessmartin** [comment]
@bgins @therealjeffg could one of you review before pod planning today?

File: README.md (hunk 1 above)
Comment (therealjeffg) [reply README.md:9]: Nope that's about right.

**Review by therealjeffg — APPROVED** [review]
This looks great - thanks! Assuming all the deployment instructions are correct, 👍🏻

File: README.md [hunk 2]
```diff
@@ -40,9 +45,11 @@ The app template includes a functioning application: an image gallery. Check out
 
 ## 🚀 Getting Started
 
-You can try out the template yourself [here](https://webnative.netlify.app/).
+You can try out the template yourself [here](https://webnative-template.fission.app/).
+
+Ready? Let's go.
 
-Ready? Let's go:
+Prerequiste: ensure you are running Node 16.14 or greater, but _not_ Node 17 (18 is fine though!).
```
Comment (bgins) [inline README.md:52]: Great idea to call this out! 💯

File: README.md (hunk 1 above)
Comment (bgins) [reply README.md:9]: The warning box isn't displaying for me when I view the rendered version here: https://github.com/webnative-examples/webnative-app-template/blob/449a28488fe21c6ea696b76684515407ba3a3da0/README.md

**Review by bgins — APPROVED** [review]
Looks great! ✨ 

One thing I noticed is the links to the Vercel and Cloudflare Pages previews are 404. Are the deploys coming in on another PR?

**Comment by jessmartin** [comment]
> One thing I noticed is the links to the Vercel and Cloudflare Pages previews are 404.

Whoops! One is a bad link, and the other (Cloudflare Pages) had to be deleted because it couldn't be "transferred" to the corporate Cloudflare account. They'll be fixed in a few minutes. 🤦‍♂️

**Pull request merged**